Assets/Scripts/UI/SettingHelper.cs

```diff
@@ -12,25 +12,133 @@ public class SettingHelper : MonoBehaviour {
 	public Toggle fpsToggle;
 	public GameObject fpsObject;
 
-	// Use this for initialization
-	void Start () {
-	
+	public Toggle fullScreenToggle;
+
+	//public Slider masterVolume;
+	public Slider musicVolume;
+
+	public Resolution[] myResolutions;
+	public Dropdown resolutionDropdown;
+
+	public Dropdown aliasingDropdown;
+	public Dropdown vSyncDropdown;
+	public Dropdown quilityDropdown;
+
+	public GameSettings gameSettings;
+
+
+	void OnEnable()
+	{
+		//create an instance of this class
+		gameSettings = new GameSettings ();
+
+		//get all avalible resolution for the display
+		myResolutions = Screen.resolutions;
+
+		//Add our listeners
+		fpsToggle.onValueChanged.AddListener(delegate { OnFPSToggle(); } );
```

Comment:
`fpsToggle.onValueChanged.AddListener(OnFPSToggle);` is shorter and should also work